addons/binding/org.openhab.binding.mcp23017/src/main/java/org/openhab/binding/mcp23017/internal/GPIODataHolder.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mcp23017.internal;
+
+import com.pi4j.gpio.extension.mcp.MCP23017Pin;
+import com.pi4j.io.gpio.GpioController;
+import com.pi4j.io.gpio.GpioFactory;
+import com.pi4j.io.gpio.Pin;
+import com.pi4j.io.gpio.PinPullResistance;
+import com.pi4j.wiringpi.GpioUtil;
+
+/**
+ * The {@link GPIODataHolder} holds a reference to GpioController.
+ * There should be only one instance per whole system
+ *
+ * @author Anatol Ogorek - Initial contribution
+ */
+public class GPIODataHolder {
+    static {
+        GpioUtil.enableNonPrivilegedAccess();
+    }
+    public static final GpioController gpio = GpioFactory.getInstance();
```

Comment:
This is a constant, please follow the naming conventions and rename it to `GPIO`